### PR TITLE
Add description of amendments to OFF clause 7.2.3

### PR DESCRIPTION
**Pull request by PeterConstable**

### CODE REVIEW

**Comment by drott** [comment]
Should fix https://github.com/MPEGGroup/OpenFontFormat/issues/33 once we update the OFF amendment doc.

File: colr-gradients-spec.md [hunk 1]
```diff
@@ -753,6 +754,136 @@ The hour hand is reusable as a transformed glyph.
 Another example might be emoji faces: many have the same backdrop
 with different eyes, noses, tears, etc drawn on top.
 
+## OFF 7.2.3 Item variation stores
+
+_Delete the fourth paragraph, "Variation data is comprised..."._
+
+_Add a new sub-clause 7.2.3.1 after the third paragraph ("The item variation store formats..."), with text as follows:_
+
+**7.2.3.1 Associatig target items to variation data**
```
Comment (anthrotype) [inline colr-gradients-spec.md:763]:
```suggestion
**7.2.3.1 Associating target items to variation data**
```

**Comment by PeterConstable** [comment]
@drott Can this get merged?

**Comment by rsheeter** [comment]
@drott @anthrotype any concerns or does this seem correct?

**Review by anthrotype — APPROVED** [review]

**Comment by rsheeter** [comment]
Merging as I see @anthrotype approved. @drott PTAL when you have a chance.

**Pull request merged**